.github/CONTRIBUTING.md

```diff
@@ -41,6 +41,7 @@ You'll see exactly what is sent, be able to add your comments, and then send it.
 
 ## Communication
 
-* The [#newpipe](irc:irc.freenode.net/newpipe) channel on freenode has the core team and other developers in it. [Click here for webchat](https://webchat.freenode.net/?channels=newpipe)!
-* You can also use a Matrix account to join the Newpipe channel at [#freenode_#newpipe:matrix.org](https://matrix.to/#/#freenode_#newpipe:matrix.org).
+* The #newpipe channel on Libera Chat (`ircs://irc.libera.chat:6697/newpipe`) has the core team and other developers in it. [Click here for webchat](https://web.libera.chat/#newpipe)!
+* You can also use a Matrix account to join the NewPipe channel at [
+#newpipe:libera.chat](https://matrix.to/#/#newpipe:libera.chat).
```

Comment:
```suggestion
* You can also use a Matrix account to join the NewPipe channel at [#newpipe:libera.chat](https://matrix.to/#/#newpipe:libera.chat).
```

Reply:
Doesn't make any difference visually, but the random line break is confusing.

Reply:
An error when I made the changes (on my phone). Thank you.